reaction-example-plugin.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Pretty sure this isn't supposed to be checked in. I think it's a Webstorm file?

Reply:
Done. going to create a new PR soon.